src/behavior.ts

```diff
@@ -114,5 +110,18 @@ export class Behavior implements IInteractiveBehavior {
                 !selected && hasHighlights
             );
         });
+
+        selection.attr("aria-selected", (dataPoint: HierarchyRectangularNode<SunburstDataPoint>) => {
+            return dataPoint.data.selected;
+        });
+
+        selection.on("keydown", (e, d: HierarchyRectangularNode<SunburstDataPoint>) => {
+            // if keydown is not enter or space, return
+            if (e.keyCode !== 13 && e.keyCode !== 32) {
```

Comment:
Use [code ](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) instead of keyCode

nit: you can add constants for codes.

Reply:
Done: [6f3e3c3](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/6f3e3c3a495dab6c497e51b353a189744d85cdc9)